### PR TITLE
[NEW] Add department_id in account.invoice.report

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

**Comment by dreispt** [comment]
LGTM :+1:
Can you use the new shorter copyright notices at the top of the fiels?

**Comment by dreispt** [comment]
LGTM, but Mr. Travis is not happy.
Can you use the new shorter copyright notices at the top of the files?

**Comment by mileo** [comment]
@dreispt Travis is green now =D

**Comment by elicoidal** [comment]
:+1:

File: invoice_department/__init__.py [hunk 1]
```diff
@@ -1 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (feketemihai) [inline invoice_department/__init__.py:1]: @mileo Can you update the headers here with the company and license..

File: invoice_department/__openerp__.py [hunk 2]
```diff
@@ -10,6 +10,9 @@
     'website': 'http://camptocamp.com',
```
Comment (feketemihai) [inline invoice_department/__openerp__.py]: The same here...short headers with company and license...

File: invoice_department/__openerp__.py [hunk 3]
```diff
@@ -10,6 +10,9 @@
...
     'license': 'AGPL-3',
     'depends': ['account', 'hr'],
-    'data': ['invoice_view.xml'],
+    'data': [
+        'invoice_view.xml',
```
Comment (feketemihai) [inline invoice_department/__openerp__.py]: Please create a new directory for views...

File: invoice_department/__init__.py [hunk 4]
```diff
@@ -1 +1,3 @@
...
 from . import invoice
```
Comment (feketemihai) [inline invoice_department/__init__.py]: Can you create a new directory for models and update the code to the new api in the function...

**Comment by feketemihai** [comment]
@mileo Thanks for the work done on this, some small remarks, can you review it, please. Thnaks again for your contribution.

**Comment by mileo** [comment]
@feketemihai Would not it be better to do this in a separate PR? (Migrating to the new API, create the README.md file, create directories, etc.)

**Comment by feketemihai** [comment]
@mileo If you use git mv for new directories, the history stays...the new api is recomended when port to v8, not neccesary...but since it's a small function,i think it's easier to do it now...if you want...

**Comment by mileo** [comment]
@feketemihai done

File: invoice_department/models/invoice.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+from openerp import models, fields, api
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    department_id = fields.Many2one(
+        comodel_name='hr.department',
+        string='Department',
+        defaut='_get_department',
```
Comment (feketemihai) [inline invoice_department/models/invoice.py]: Missed an "l"...default...

Comment (mileo) [inline invoice_department/models/invoice.py]: Sorry

**Comment by feketemihai** [comment]
@mileo Can you also update copyrigths, i don't see them on old files done by camptocamp...even if it's out of the scope of the pr...

**Comment by mileo** [comment]
@feketemihai Can i update with https://github.com/OCA/department/commit/15e23e4c4b8590ca5200d343e7e47931a8a005bf ?
  # Copyright (c) 2011 Camptocamp SA (http://www.camptocamp.com)
  # Author : Joel Grand-guillaume (Camptocamp)

**Comment by feketemihai** [comment]
@mileo only copyrigths..authors goes in a separate tab in readme, Contributors...you should add yourself overthere...

**Comment by mileo** [comment]
@feketemihai I believe that now is all right. I am available to fix anything else.

**Comment by feketemihai** [comment]
@mileo Thaks for quick fixes...only one think...can you add the Usage tab in readme with link to Try me on Runbot, otherwise :+1: .

Thanks for you contribution.

**Comment by mileo** [comment]
@feketemihai 
ok, I will look at it and travis error

**Comment by dreispt** [comment]
@mileo Can you check TravisCI please?

**Comment by mileo** [comment]
@dreispt Yes, I'll fix it this weekend

**Comment by mileo** [comment]
Fixed

**Comment by feketemihai** [comment]
:+1:

**Comment by dreispt** [comment]
:+1:

**Pull request merged**